app/controllers/veterans_controller.rb

```diff
@@ -70,10 +70,14 @@ def download_all_veterans
   def word
       @for_fed_employment = params[:fed]=="true"
       respond_to do |format|
-        format.html do
+        format.docx do
           response.headers['Content-Disposition'] = 'attachment; filename="resume.doc"'
           render content_type: "application/vnd.openxmlformats-officedocument.wordprocessingml.document", layout: 'word'
         end
+        format.html do
```

Comment:
Is this ever called?  why does the federal resume return in html format but the non-federal one in docx?